### PR TITLE
Release `2024.8.2`

### PR DESCRIPTION
**Pull request by PeterNerlich**
# Release notes

- [ [#2007](https://github.com/digitalfabrik/integreat-cms/issues/2007) ] Bulk action for canceling translation process
- [ [#2797](https://github.com/digitalfabrik/integreat-cms/issues/2797) ] Fix automatic url update after changing a title of the page on side by side view
- [ [#2801](https://github.com/digitalfabrik/integreat-cms/issues/2801) ] Hide links to images with empty alt attributes
- [ [#2807](https://github.com/digitalfabrik/integreat-cms/issues/2807) ] Fix a bug where machine-translated pages where considered out-of-date
- [ [#2874](https://github.com/digitalfabrik/integreat-cms/issues/2874) ] Retry mechanism for push notifications that were intended to be sent immediately but failed due to an error
- [ [#2969](https://github.com/digitalfabrik/integreat-cms/issues/2969) ] The translation report shows all language labels on the y-axis
- [ [#2982](https://github.com/digitalfabrik/integreat-cms/issues/2982) ] Include links in imprints for link replacement
- [ [#2989](https://github.com/digitalfabrik/integreat-cms/issues/2989) ] Fix bug where adding an event on the last weekday of every month only adds it in months with 5 weeks
- [ [#2998](https://github.com/digitalfabrik/integreat-cms/issues/2998) ] Ignore soft hyphens when detecting links
- [ [#3001](https://github.com/digitalfabrik/integreat-cms/issues/3001) ] Fix link auto update checkbox always checked
- [ [#3005](https://github.com/digitalfabrik/integreat-cms/issues/3005) ] Add go back link for POIS and events

### CODE REVIEW

**Review by JoeyStk — CHANGES_REQUESTED** [review]
The recurrence rule is sadly still broken and needs to be fixed, before we can release a new version.
Blocked by #3022

**Comment by PeterNerlich** [comment]
I plan to add a commit on `develop` directly to address the release notes affected here by #3024 before merging this release.

**Review by JoeyStk — DISMISSED** [review]
The only issue I had was right at the beginning. I had to use `prune_database.sh` to fix a JSONDecodeError. But as far as I can tell this was an issue with my local setup - but maybe it's something to keep an eye on :)

**Pull request merged**
